rllib/policy/view_requirement.py

```diff
@@ -48,12 +50,29 @@ def __init__(self,
                 Example: For a view column "obs" in an Atari framestacking
                 fashion, you can set `data_col="obs"` and
                 `data_rel_pos=[-3, -2, -1, 0]`.
+                Example: For the obs input to an attention net, you can specify
+                a range via a str: `data_rel_pos="-100:0"`, which will pass in
+                the past 100 observations plus the current one.
+            abs_pos (Optional[int]): An optional absolute position arg,
+                used e.g. for the location of a requested inference dict within
+                the trajectory. Negative values refer to counting from the end
+                of a trajectory.
             used_for_training (bool): Whether the data will be used for
                 training. If False, the column will not be copied into the
                 final train batch.
+            is_input_dict (bool): Whether the "view" of this requirement is an
```

Comment:
This seems odd, what is the reason we need it? Are there any cleaner alternatives?

Reply:
This option will be necessary for attention nets. We shouldn't have attention net or RNN-specific code in the postprocessing fn (e.g. of PPO). Instead, it's like saying: "I need an input_dict here, provide one given the model's requirements".